content/blog/2022-05-17-aws-remotes-in-dvc.md

```diff
@@ -0,0 +1,170 @@
+leave the default settings in place or you can update them to fit the
+functionality you need.
+
+![create an S3 bucket in AWS](/uploads/images/2022-05-17/create_bucket.png)
+
+Once you’ve created the bucket, you should be redirected to the S3 dashboard and
+see the success message and your new bucket.
+
+![newly created S3 bucket in AWS](/uploads/images/2022-05-17/created_bucket.png)
+
+### Get your credentials
+
+Now that the S3 bucket is ready, we need the `access_key_id` and
+`secret_access_key` from AWS in order to connect to our project. You can create
+these keys in your Identity and Access Management settings. Go to your security
+credentials and select the `Access keys` section. Then click the
+`Create New Access Key` button. This will generate a new set of keys for you so
+make sure you download this file to get your secret access key.
+
+![make AWS access credentials](/uploads/images/2022-05-17/make_credentials.png)
+
+Once you’ve downloaded the credentials, you should see the access key ID in the
+table. Note that you won’t be able to access your secret key again at this
+point. You would need to make a new set of credentials if you don’t have it.
+
+![successfully created AWS access credentials](/uploads/images/2022-05-17/credentials.png)
+
+That’s it for setting up your bucket and getting the credentials you need! Now
+let’s add DVC to our demo repo and set up the remote.
+
+## Set up a DVC project
+
+First, add DVC as a requirement to your project with the following installation
+command:
+
+```dvc
+$ pip install 'dvc[s3]'
+```
+
+Then you can initialize DVC in your own project with the following command:
+
+```dvc
+$ dvc init
+```
+
+This will add all of the DVC internals needed to start versioning your data and
+tracking experiments. Now we need to set up the remote to connect our project
+data stored in AWS to the DVC repo.
+
+### Create a default remote
+
+Now we can add a default to the project with the following command:
+
+```dvc
+$ dvc remote add -d bikes s3://updatedbikedata
+```
+
+This creates a default remote called `bikes` that connects to the
+`updatedbikedata` bucket we made earlier which is where the training data for
+the model will be stored.
+
+### Add AWS credentials
+
+In order for DVC to be able to push and pull data from the remote, you need to
+have valid AWS credentials.
+
+By default, DVC authenticates using your AWS CLI configuration, if it has been
+set. You can check that out
+[here in the AWS docs](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
+
+If you want to
+[use a different authentication method](https://dvc.org/doc/command-reference/remote/modify#amazon-s3)
+or if you run into issues with the credentials, you can manually add them with
+the following commands:
+
+```dvc
+$ dvc remote modify --local bikes access_key_id 'mykey'
+```
+
+```dvc
+$ dvc remote modify --local bikes secret_access_key 'mysecret'
+```
```

Comment:
Do we ever use hidden sections or offset text within a blog post? It seems like it might be useful to indicate that these steps aren't necessary to follow along.

Reply:
BTW no need for 2 separate blocks
````suggestion
If you want to
[use a different authentication method](https://dvc.org/doc/command-reference/remote/modify#amazon-s3)
or if you run into issues with the credentials, you can manually add them with
the following commands:

```dvc
$ dvc remote modify --local bikes access_key_id 'mykey'
$ dvc remote modify --local bikes secret_access_key 'mysecret'
```
````

Reply:
@dberenbaum We haven't done a hidden section in the blog before. It looks like the styles aren't the same as in the docs so I'll leave it out.

<img width="795" alt="Screen Shot 2022-05-17 at 09 04 15" src="https://user-images.githubusercontent.com/47196133/168829886-e2b4b845-404b-4454-b37e-1555d36a745e.png">